src/main/kotlin/dev/stalla/util/ExperimentalAPI.kt

```diff
@@ -0,0 +1,20 @@
+package dev.stalla.util
+
+/**
+ * Annotation class indicating an experimental API that should not be relied on yet.
+ *
+ * @since 1.0.0
+ */
+@RequiresOptIn(
+    level = RequiresOptIn.Level.WARNING,
+    message = "This API is experimental. It may change or behave different in future releases."
+)
+@Target(
+    AnnotationTarget.CLASS,
```

Comment:
I would sort these alphabetically; also just noticed we're missing some potential targets: `LOCAL_VARIABLE`, `VALUE_PARAMETER`, and `PROPERTY_GETTER`. Local variables are somewhat unlikely to need this annotation, but I'd include them anyway.

Same for `@InternalAPI`.

Reply:
I have to admit, I just copied the targets from Ktor without checking if they are complete 😅 . You are totally right of course, we are missing targets!

Reply:
Was a stroke of luck, really! :)

Reply:
`@InternalAPI` does have `PROPERTY_SETTER`, fwiw